blueprints/flexberry-edit-form/files/__root__/routes/__name__/new.js

```diff
@@ -2,5 +2,6 @@ import EditFormNewRoute from 'ember-flexberry/routes/edit-form-new';
 
 export default EditFormNewRoute.extend({
   modelProjection: '<%=modelProjection%>',
-  modelName: '<%=modelName%>'
+  modelName: '<%=modelName%>',
+  templateName: '<%=entityName%>',
```

Comment:
I consulted with a specialist.
I thought about using 'formName', this name form of metadata from designer.